### PR TITLE
Update package.json WSL IP for Windows users

### LINKED ISSUE #8: Journey to automatically update package.json local ip address
### What would you like to change?
Create a new journey for the package.json files of EnglandWales and Scotland to update with the IP address correctly inserted into config.local.et_cos as "http://[IP adderss]:8081".

### How do you think that would improve the project?
Developers that use Windows with WSL need to manually update their WSL IP address every time WSL restarts (every time Windows restarts or when wsl is restarted). This would make the process slightly faster and less error prone.

### PR DESCRIPTION
**Pull request by Itay-Kirsh**
### Change description ###
Implement journey that updates json.config.local.et_cos with WSL IP for Windows users.
Fix #8 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: journeys/et/updateIP.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+async function updateIP() {
+  const ip = (await execCommand("hostname -I | awk '{print $1}'")).stdout.trim();
+  updateRegion(process.env.ENGWALES_DEF_DIR, ip)
+  updateRegion(process.env.SCOTLAND_DEF_DIR, ip)
+}
+
+function updateRegion(regionDir: string, ip: string) {
+  const packageJsonPath = `${regionDir}${sep}package.json`;
+
+  const json = JSON.parse(readFileSync(packageJsonPath).toString())
```
Comment (JackReevies) [inline journeys/et/updateIP.ts:15]: should be able to use require here, it'll parse json automatically

File: journeys/et/updateIP.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  // noinspection HttpUrlsUsage
```
Comment (JackReevies) [inline journeys/et/updateIP.ts:16]: Guessing this is an intelliJ specific thing? vscode doesnt seem to complain without it

File: journeys/et/updateIP.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import {Journey} from "types/journey"
+import {readFileSync, writeFileSync} from "fs";
+import {sep} from "path";
+import {execCommand} from "app/helpers";
```
Comment (JackReevies) [inline journeys/et/updateIP.ts:4]: We should probably look into officially enforcing some styling rules, personally I prefer spaces between the imports (like `import { execCommand } from....`. I usually use "StandardJS" but I'll admit its not setup in my local environment, maybe we should?

Also get rid of semicolons [JS doesn't need them (and if you run into a case where it does, its not good code)](https://standardjs.com/rules.html#semicolons)

File: journeys/et/updateIP.ts (hunk 1 above)
Comment (Itay-Kirsh) [reply journeys/et/updateIP.ts:15]: What do you mean by require?

**Review by JackReevies — CHANGES_REQUESTED** [review]
Looks good! Just a few nit picks. Also would `dockerUpdateIP` be a better name for the file? I see its in the `et-docker` group, would be nice to keep it with the other `et-docker` files

File: journeys/et/updateIP.ts (hunk 1 above)
Comment (JackReevies) [reply journeys/et/updateIP.ts:15]:
```
  const packageJsonPath = `${regionDir}${sep}package.json`
  const packageJson = require(packageJsonPath)

  packageJson.config.local.et_cos = `http://${ip}:8081`
  writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2))
 ```

File: journeys/et/updateIP.ts (hunk 2 above)
Comment (Itay-Kirsh) [reply journeys/et/updateIP.ts:16]: Yep

File: journeys/et/updateIP.ts (hunk 3 above)
Comment (Itay-Kirsh) [reply journeys/et/updateIP.ts:4]: Good idea to set something like that up. I'll fix it for now.

File: journeys/et/updateIP.ts (hunk 2 above)
Comment (JackReevies) [reply journeys/et/updateIP.ts:16]: Is there a way to disable it that doesnt involve adding comments in code? Would like to avoid that where possible

Comment (Itay-Kirsh) [reply journeys/et/updateIP.ts]: Yep, will do, probably nicer.

**Review by JackReevies — DISMISSED** [review]

**Review by JackReevies — APPROVED** [review]

**Pull request merged**